mne/tests/test_utils.py

```diff
@@ -104,6 +104,29 @@ def test_object_size():
                     msg='%s < %s < %s:\n%s' % (lower, size, upper, obj))
 
 
+def test_get_inst_data():
+    from mne.epochs import _segment_raw
+    from mne.time_frequency import tfr_morlet
+
+    raw = read_raw_fif(fname_raw)
+    # chns = raw.ch_names[:5]
+    # raw.pick_channels(chns)
```

Comment:
Comaring to a slow test isn't a great comparison :) If it's less than, say, 0.1 sec don't worry about it. If it's longer yes please try to shorten.